### PR TITLE
#nogaun/issue_0 Registration

### PR DESCRIPTION
**Pull request by duythinh96**
My name is NoGaUn
Please help me review this pull and accept me to this repo
Thank you very much 🙇

### CODE REVIEW

File: public/nogaun/index.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    <title>NoGaUn</title>
+    <link rel="stylesheet" href="../../style.css">
+</head>
+<body>
+<h1>Welcome to NoGaUn page</h1>
+<div class="flex">
+    <a href="#">#0: Registration</a>
+</div>
+</body>
+</html>
```
Comment (timngac) [inline public/nogaun/index.html]: Please add break line for this file

File: public/nogaun/issue_1/index.html [hunk 2]
```diff
@@ -0,0 +1 @@
+Registration
```
Comment (timngac) [inline public/nogaun/issue_1/index.html]: Please add break line for this line

**Review by timngac — CHANGES_REQUESTED** [review]
Please check this 👍

**Comment by duythinh96** [comment]
Please help me check again 🙇

**Review by timngac — CHANGES_REQUESTED** [review]
That good
Please add class `done` to issue 0 in your html.index file

**Review by timngac — APPROVED** [review]
I will merge this pull, please wait 👍

**Pull request merged**